activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/ErrorAuditProducerIT.java

```diff
@@ -0,0 +1,146 @@
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@ActiveProfiles(AuditProducerIT.AUDIT_PRODUCER_IT)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestPropertySource("classpath:application-test.properties")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+public class ErrorAuditProducerIT {
+
+    private static final String ERROR_START_EVENT_SUBPROCESS = "errorStartEventSubProcess";
+    
+    @Autowired
+    private AuditConsumerStreamHandler streamHandler;
+    
+    @Autowired
+    private ProcessInstanceRestTemplate processInstanceRestTemplate;
+
+    @Before
+    public void setUp() {
+        streamHandler.clear();
+    }
+
+    @Test
+    public void should_produceBpmnErrorEvents_when_processIsExecuted() {
+        
+        ResponseEntity<CloudProcessInstance> startProcessEntity = processInstanceRestTemplate.startProcess(new StartProcessPayloadBuilder()
+                                                                                                            .withProcessDefinitionKey(ERROR_START_EVENT_SUBPROCESS)
+                                                                                                            .withName("processInstanceName")
+                                                                                                            .withBusinessKey("businessKey")
+                                                                                                            .build());
+
+        assertThat(startProcessEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
+        
+        CloudProcessInstance processInstance = startProcessEntity.getBody();
+
+        await().untilAsserted(() -> {
+            assertThat(streamHandler.getReceivedHeaders()).containsKeys(RUNTIME_BUNDLE_INFO_HEADERS);
+            assertThat(streamHandler.getReceivedHeaders()).containsKeys(ALL_REQUIRED_HEADERS);
+            List<CloudRuntimeEvent<?, ?>> receivedEvents = streamHandler.getAllReceivedEvents();
+
+            assertThat(receivedEvents)
+            .filteredOn(event -> (event.getEventType().equals(ACTIVITY_STARTED) ||
+                                  event.getEventType().equals(ACTIVITY_COMPLETED)))
+            .extracting(CloudRuntimeEvent::getEventType,
+                        event -> ((BPMNActivity) event.getEntity()).getActivityType(),
+                        event -> ((BPMNActivity) event.getEntity()).getElementId(),
+                        event -> ((BPMNActivity) event.getEntity()).getProcessInstanceId())
+            .contains(tuple(ACTIVITY_STARTED,
```

Comment:
I'm approving to unblock tests, but we need to investigate why only `ACTIVITY_STARTED` is available for end event. I'd expect to have `ACTIVITY_COMPLETED` as well.